ppc64le:  spack install xextproto fails due to old config.guess and config.sub files

I"m trying to build spack on a power 8 cluster and I'm getting the error below due to the old config.guess and config.sub files that don't have information about the ppc64le platform.

The source for this package hasn't been touched for several years.
I guess the only way to fix this is to create a patch that replaces the config.guess and config.sub configuration files.

I can do the patch, if people agree and there is not another fix that I'm not seeing.
Thanks,
Jim G
```
cat -n /tmp/jeg/spack-stage/spack-stage-82EJxv/xextproto-7.3.0/spack-build.out

   44	This script, last modified 2009-12-30, has failed to recognize
    45	the operating system you are using. It is advised that you
    46	download the most up to date version of the config scripts from
    47	
    48	  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
    49	and
    50	  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD
    51	
    52	If the version you run (/home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o/xextproto-7.3.0/config.guess) is already up to date, please
    53	send the following data and any information you think might be
    54	pertinent to <config-patches@gnu.org> in order to provide the needed
    55	information to handle your system.
    56	
    57	config.guess timestamp = 2009-12-30
    58	
    59	uname -m = ppc64le
    60	uname -r = 3.10.0-514.21.2.el7.ppc64le
    61	uname -s = Linux
    62	uname -v = #1 SMP Tue Jun 20 16:35:48 GMT 2017
    63	
    64	/usr/bin/uname -p = ppc64le
    65	/bin/uname -X     = 
    66	
    67	hostinfo               = 
    68	/bin/universe          = 
    69	/usr/bin/arch -k       = 
    70	/bin/arch              = ppc64le
    71	/usr/bin/oslevel       = 
    72	/usr/convex/getsysinfo = 
    73	
    74	UNAME_MACHINE = ppc64le
    75	UNAME_RELEASE = 3.10.0-514.21.2.el7.ppc64le
    76	UNAME_SYSTEM  = Linux
    77	UNAME_VERSION = #1 SMP Tue Jun 20 16:35:48 GMT 2017
    78	configure: error: cannot guess build type; you must specify one
```

@lee218llnl This is what I'm seeing:
See build log for details:
  /tmp/jeg/spack-stage/spack-stage-bVndsS/xextproto-7.3.0/spack-build.out
[jeg@p8-node packages]$ rm -rf /home/jeg/spack/var/spack/cache/xextproto/xextproto-7.3.0.tar.gz
[jeg@p8-node packages]$ rm -rf /home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o/xextproto-7.3.0.tar.gz
[jeg@p8-node packages]$ rm -rf /home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o
[jeg@p8-node packages]$ spack install openspeedshop
==> flex is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/flex-2.6.4-skx72if6qe7lfx7fxkrsv4xze5yqfmme
==> libtool is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/libtool-2.4.6-37ahgf7zyhrf7pumvrw4knfrhhyrhc6g
==> cmake is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/cmake-3.9.0-tyxqnvgjtjhmfnorho6mqakfmgugft4r
==> freetype is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/freetype-2.7.1-3thidjv3yxs7ric2arus3to2exnnolss
==> kbproto is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/kbproto-1.0.7-zfv74oeykbkkvulxx2t7lyjw5duc66jz
==> util-macros is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/util-macros-1.19.1-imyaktcwkmm6opd3w4gh6kmbptury5bs
==> pkg-config is already installed in /home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/pkg-config-0.29.2-la6vhvtxv46nablpxullwu5ek6rxrld4
==> Installing xextproto
==> Fetching https://www.x.org/archive/individual/proto/xextproto-7.3.0.tar.gz
######################################################################## 100.0%
==> Staging archive: /home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o/xextproto-7.3.0.tar.gz
==> Created stage in /home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o
==> No patches needed for xextproto
==> Building xextproto [AutotoolsPackage]
==> Executing phase: 'autoreconf'
==> Executing phase: 'configure'
==> Error: ProcessError: Command exited with status 1:
    '/home/jeg/spack/var/spack/stage/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o/xextproto-7.3.0/configure' '--prefix=/home/jeg/spack/opt/spack/linux-centos7-ppc64le/gcc-4.8.5/xextproto-7.3.0-v6t66lknzc5gb66hy5guqvraldd7ql6o'

1 errors in build log:
     [ ... ]
     74    /usr/convex/getsysinfo =
     75    
     76    UNAME_MACHINE = ppc64le
     77    UNAME_RELEASE = 3.10.0-514.21.2.el7.ppc64le
     78    UNAME_SYSTEM  = Linux
     79    UNAME_VERSION = #1 SMP Tue Jun 20 16:35:48 GMT 2017
  >> 80    configure: error: cannot guess build type; you must specify one

See build log for details:
  /tmp/jeg/spack-stage/spack-stage-fyjQfS/xextproto-7.3.0/spack-build.out
[jeg@p8-node packages]$

--- a/lib/spack/spack/build_systems/autotools.py
+++ b/lib/spack/spack/build_systems/autotools.py
@@ -103,7 +103,7 @@ class AutotoolsPackage(PackageBase):
         build date (automake 1.13.4)."""
 
         if not self.patch_config_guess or not self.spec.satisfies(
-                'arch=linux-rhel7-ppc64le'
+                'target=ppc64le'
         ):
             return
         my_config_guess = None

